### PR TITLE
{bio,lib}[GCCcore/8.2.0] pyEGA3 v3.0.33, tqdm v4.32.1 w/ Python 3.7.2

### PR DESCRIPTION
**Pull request by smoors**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
nic427 - Linux centos linux 7.5.1804, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/579a7b8137b2adb977bf5731d84aa184 for a full test report.

File: easybuild/easyconfigs/t/tqdm/tqdm-4.32.1-GCCcore-8.2.0-Python-3.7.2.eb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+toolchain = {'name': 'GCCcore', 'version': '8.2.0'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['0a860bf2683fdbb4812fe539a6c22ea3f1777843ea985cb8c3807db448a0f7ab']
+
+builddependencies = [('binutils', '2.31.1')]
+
+dependencies = [('Python', '3.7.2')]
```
Comment (akesandgren) [inline easybuild/easyconfigs/t/tqdm/tqdm-4.32.1-GCCcore-8.2.0-Python-3.7.2.eb]: Can this be done with multi_deps for both python 2 and 3?
Same for pyEGA3

Comment (smoors) [reply easybuild/easyconfigs/t/tqdm/tqdm-4.32.1-GCCcore-8.2.0-Python-3.7.2.eb]: @akesandgren pyEGA3 works only with python3

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
nic418 - Linux centos linux 7.5.1804, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/b94985d318bdc67ed03963777078fd10 for a full test report.

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
cermis - Linux debian 9.9, Intel(R) Xeon(R) CPU           X5670  @ 2.93GHz, Python 2.7.13
See https://gist.github.com/6998e4559a3fb2c7bf369204dc31f313 for a full test report.

**Comment by vanzod** [comment]
Going in, thanks @smoors!

**Pull request merged**